server/routes/serviceProviderReferrals/postSessionBehaviourFeedbackPresenter.ts

```diff
@@ -0,0 +1,40 @@
+import { DeliusServiceUser } from '../../services/communityApiService'
+import { ActionPlanAppointment } from '../../services/interventionsService'
+import PresenterUtils from '../../utils/presenterUtils'
+import ServiceUserBannerPresenter from '../shared/serviceUserBannerPresenter'
+
+export default class PostSessionBehaviourFeedbackPresenter {
+  constructor(
+    private readonly appointment: ActionPlanAppointment,
+    private readonly serviceUser: DeliusServiceUser,
+    private readonly userInputData: Record<string, unknown> | null = null
+  ) {}
+
+  readonly text = {
+    title: `Add behaviour feedback`,
+    behaviourDescription: {
+      question: `Describe ${this.serviceUser.firstName}'s behaviour in this session`,
+      hint: 'For example, consider how well-engaged they were and what their body language was like.',
+    },
+    notifyProbationPractitioner: {
+      question: 'If you described poor behaviour, do you want to notify the probation practitioner?',
+      explanation: 'If you select yes, the probation practitioner will be notified by email.',
+      hint: 'Select one option',
+    },
+  }
+
+  readonly serviceUserBannerPresenter = new ServiceUserBannerPresenter(this.serviceUser)
+
+  private readonly utils = new PresenterUtils(this.userInputData)
+
+  readonly fields = {
+    behaviourDescriptionValue: new PresenterUtils(this.userInputData).stringValue(
```

Comment:
If you want to be consistent with the field below, I think this should be called `behaviourDescription`.